db/structure.sql

```diff
@@ -3914,6 +3914,7 @@ CREATE MATERIALIZED VIEW public.reporting_overdue_patients AS
     patient_with_call_results_and_phone.previous_call_result_type,
     patient_with_call_results_and_phone.previous_call_removed_from_overdue_list_reason,
         CASE
+            WHEN (patient_with_call_results_and_phone.previous_appointment_id IS NULL) THEN 'no'::text
```

Comment:
migration file is adding a bunch of indexes, but they aren't showing in structure.sql. Is this correct behaviour?

Reply:
Those migrations already exist on the `reporting_overdue_patients` table. Hence don't need to add them again.

Reply:
> Those indices already exist on the `reporting_overdue_patients` table. Hence don't need to add them again.

`reporting_facility_states` *